core/whistle-1.0.0/src/wh_cache.erl

```diff
@@ -602,7 +602,11 @@ maybe_erase_changed(Db, 'database', _Id, Tab) ->
     erase_changed(Objects, [], Tab);
 
 maybe_erase_changed(Db, Type, Id, Tab) ->
-    MatchSpec = [{#cache_obj{origin = {'db', Db}, _ = '_'}
+    MatchSpec = [{#cache_obj{origin = {'database', Db}, _ = '_'}
```

Comment:
yes, for instance, https://github.com/2600hz/kazoo/blob/master/core/whistle_apps-1.0.0/src/whapps_util.erl#L400-L405

seems wrong to me, we are querying accounts db but caching the accountdb retrieved.
in this case a simple {'db', ?WH_ACCOUNTS, <<"account">>} is more correct because it would also invalidate when a new account is created which it doesn't right now